cmake/bintools/arcmwdt/target.cmake

```diff
@@ -22,17 +22,18 @@ find_program(CMAKE_GDB     ${CROSS_COMPILE}mdb     PATHS ${TOOLCHAIN_HOME} NO_DE
 
 # MWDT binutils don't support required features like section renaming, so we
 # temporarily had to use GNU objcopy instead
-find_program(CMAKE_OBJCOPY arc-elf32-objcopy)
-if (NOT CMAKE_OBJCOPY)
-  find_program(CMAKE_OBJCOPY arc-linux-objcopy)
+if(CONFIG_ISA_ARCV3)
+  set(ARCMWDT_Z_HELPER_TARGET       arc64-zephyr-elf)
+else()
+  set(ARCMWDT_Z_HELPER_TARGET         arc-zephyr-elf)
```

Comment:
This actively prevents using gnu tools that are not part of sdk-ng

Reply:
The current solution actively prevents using tools part of sdk-ng. Since we have the SDK (and users need it anyway for host tools), why not make it the default?

Reply:
Default is fine, forcing it to only work on those tools is not OK. As per https://docs.zephyrproject.org/latest/develop/toolchains/index.html there is no requirement for users to have sdk-ng installed to build unless specific requirements are needed, e.g. qemu.

Reply:
Not all tools required for building Zephyr are part of the toolchain. We e.g. need DT tools and binutils. I don't see an issue with requiring users to install Zephyr SDK instead of installing another ARC GNU toolchain and/or other host tools.
The 'default' could still be changed by users by editing this file or adding another toolchain that is a combination of other tools.

Reply:
There is nothing preventing third parties from building `arc-zephyr-elf` toolchain, so this change is not requiring Zephyr SDK per se.

It is reasonable for Zephyr to require a toolchain that is specifically built for Zephyr -- for instance, you would not expect *-darwin-gcc to work on Linux.